pom.xml

```diff
@@ -54,22 +55,27 @@
         <dependency>
             <groupId>org.jenkins-ci.test</groupId>
             <artifactId>docker-fixtures</artifactId>
-            <version>1.8</version>
+            <version>1.11</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-slaves</artifactId>
-            <version>1.30.0</version>
+            <version>1.31.5</version> <!-- TODO: Remove version when core is 2.277 or greater (included in BOM) -->
```

Comment:
used 1.31.5 since that is the version that is used in `bom-2.277`

Reply:
I think 1.31.5 is the version picked up by version 876.vc43b4c6423b6 of the 2.249.x line of the BOM already, so you should be able to drop this line. [`bom-2.289.x` uses 1.32.0](https://github.com/jenkinsci/bom/blob/c43b4c6423b69ddffd9b9aa477af031236d495a0/bom-latest/pom.xml#L356), and [`bom-2.277.x` inherits from `bom-2.289.x` but overrides the version to 1.31.5](https://github.com/jenkinsci/bom/blob/c43b4c6423b69ddffd9b9aa477af031236d495a0/bom-2.277.x/pom.xml#L33), and then 2.263.x inherits that same version from 2.277.x and 2.249.x inherits that version from 2.263.x.

Reply:
removed in 393fcc5eb12afdb0f08dd39e4813c4433a254678